### PR TITLE
Adding the translation to brazilian portuguese for authorization

### PR DESCRIPTION
**Pull request by jralmaraz**
This PR translates the 'content/en/docs/reference/access-authn-authz/authorization.md' page to PT-BR. 

Some links might be broken given the referenced pages have not been translated yet. For that matter: should the link be kept to the 'EN' doc until the translated links are available ? 

Thanks!

### CODE REVIEW

**Comment by stormqueen1990** [comment]
> For that matter: should the link be kept to the 'EN' doc until the translated links are available ?

Hello there, @jralmaraz!

Yes, we keep the links pointing to the original English docs until the pages get translated. I noticed you have some references from this page in your PR #34097. In this case, we can hold that until this one gets merged.

/cc

**Comment by jralmaraz** [comment]
> > For that matter: should the link be kept to the 'EN' doc until the translated links are available ?
> 
> Hello there, @jralmaraz!
> 
> Yes, we keep the links pointing to the original English docs until the pages get translated. I noticed you have some references from this page in your PR #34097. In this case, we can hold that until this one gets merged.
> 
> /cc

Thanks for confirming that and the suggestions too! I'm removing some links already then.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:32]: Typo/missing translation in "modules".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:35]: How do you feel about rephrasing "Se todos os módulos de autorização não tiverem..." negatively, i.e. "se nenhum módulo de autorização tiver..."?
It would sound more idiomatic.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:46]: "endpoints sem recursos" is sounding a little bit strange. Some suggestions:
* "endpoints que não retornam recursos".
* "endpoints que não manipulam recursos".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:49]: `--` is not usually used in Portuguese, so we can substitute it for `-`. In this case, the following "Para" should start with a lowercase "p".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:52]: In this sentence, "core" is the name of the API group. "Grupo de API _core_" would be delivering the original meaning.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:58]: Maybe "requisições que não se referem a recursos", or "requisições sem recursos" would be a better fit here.

This formatting is also slightly inconsistent with the next item (title in a separate line).

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:70]: Suggestion: use "observar" in lieu of "assistir".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:74]: Missing a "caution" admonition here.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:82]: I think we should avoid translating "core" in this context (as the name of the API group is verbatim "core").

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:88]: It's OK to not format "kubelet" or "Pod" as code. It's better to render Pod with a capital P, as a convention to refer to Kubernetes resources.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:90]: Suggestion: "recursos computacionais ou de rede" instead of "recursos de computador ou rede".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:91]: Minor: missing translation for group.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:93]: The sentence "quando ocorrerem determinadas coisas" is sounding a bit strange. Some suggestions to replace it:
* "em certas situações".
* "quando um determinado evento ocorrer".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:124]: This link is broken because the anchor was not maintained upon translation. The correct anchor would be`#personificação-de-usuário`. You can also use the term "personificação de usuário" in lieu of "user impersonation" to stay consistent with the translated version.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:159]: Minor: duplicated bullet point here.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:157]: Typo: "em este grupo".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:163]: Suggestion: remove capital initials from "Interfaces de Usuário".
Minor: "mostrarem" would be more correct as it refers to "interfaces de usuário".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:206]: Minor: "API do Kubernetes".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:218]: This sentence is reading a bit strange. Some suggestions:
* "por exemplo, um operador, e são capazes de escalar seus próprios privilégios".
* "por exemplo, um operador, e conseguiriam escalar seus próprios privilégios".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:228]: "segredo" refers to the Secret resource here. It would be better to render it non-translated, with a capital S.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:230]: Missing accent in "privilegio".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:233]: This sentence is reading incomplete. Suggestion: add "acesso" at the end.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:241]: Missing accent in "areas".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:245]: This header is broken. The original Hugo shortcode must be maintained here (`heading "whatsnext"`).

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hello there, @jralmaraz!

Thank you very much for your contribution. 🎉 

I found some issues that need to be addressed. If you need any clarification please feel free to reach out.

**Comment by jralmaraz** [comment]
Thanks a lot @stormqueen1990 . I'll go through the findings and hopefully push the fixes this week!

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (jralmaraz) [reply content/pt-br/docs/reference/access-authn-authz/authorization.md:2]: I just realized that some of the titles are shorter and thought that Authentication and Authorization being in one word would be easier. Should I stick with "Visão Geral" ? Can do that :)

**Comment by edsoncelio** [comment]
Hi @jralmaraz there are some suggestions/fixes made by @stormqueen1990 that are missing, after that it's lgtm from my side, thanks!

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (jralmaraz) [reply content/pt-br/docs/reference/access-authn-authz/authorization.md:30]: True that, my bad!

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (jralmaraz) [reply content/pt-br/docs/reference/access-authn-authz/authorization.md:35]: Perfect, thanks~!

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (jralmaraz) [reply content/pt-br/docs/reference/access-authn-authz/authorization.md:93]: thanks!

**Comment by jralmaraz** [comment]
Thanks @edsoncelio and sorry @stormqueen1990 , I left plenty of items left. It should have been updated with the latest.

Cheers!

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [reply content/pt-br/docs/reference/access-authn-authz/authorization.md:51]: Hi @jralmaraz, I think the change here is missing from the latest version (still reads "designa o core grupo de API").

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [reply content/pt-br/docs/reference/access-authn-authz/authorization.md]: It's best to stick to the original title, which in this case would amount to "Visão Geral de Autorização".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (jralmaraz) [reply content/pt-br/docs/reference/access-authn-authz/authorization.md]: Fixed on b53e77805e

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:84]: Nit: these bullet points are inverting the term "verbo" and the verb itself (in Portuguese, it should be "verbo `use`", "verbos `bind` e `escalate`, etc.).

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:90]: The terms kubelets and Pods here need not have the code formatting, as per our standards for the Portuguese localization.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:168]: Perhaps "onde a resposta no campo `status` do objeto retornado" would sound more natural?

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:208]: I'm curious about the square brackets here. In the original version, there are no links in these bullet points.

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:220]: Dangling "e" before "conseguiriam".

File: content/pt-br/docs/reference/access-authn-authz/authorization.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/access-authn-authz/authorization.md:239]: For consistency, I think all "mounting" should be translated to "montagem" and "using" could be translated to "uso de" instead.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hello there, @jralmaraz!

Thank you very much for your contribution! 🎉 

I found a few small issues left. As soon as those are fixed I'll leave a LGTM here.

Thanks again!

**Comment by stormqueen1990** [comment]
/lgtm

@edsoncelio could you take a look?

**Comment by edsoncelio** [comment]
Thank you so much @jralmaraz and @stormqueen1990!
LGTM
/approve

**Pull request merged**
